source/api/ut_runner.pks

```diff
@@ -33,31 +33,34 @@ create or replace package ut_runner authid current_user is
   function version_compatibility_check( a_requested varchar2, a_current varchar2 := null ) return integer;
 
   /**
-  * Run suites/tests by path
-  * Accepts value of the following formats:
-  * schema - executes all suites in the schema
-  * schema:suite1[.suite2] - executes all items of suite1 (suite2) in the schema.
+  * Execute specified suites/tests by paths
+  * @param a_paths                  list of schemes, packages, procedures or suite-paths to execute
```

Comment:
The “at” sign will cause problems in some environments (for example PL/SQL Developer)
Do you mind it changing to “%”?

Reply:
Oh! Didn't know. I always use at in description because it works with plsql doc...